tfjs-backend-webgpu/src/matmul_small_output_size_webgpu.ts

```diff
@@ -143,7 +105,7 @@ export class MatMulSmallOutputSizeProgram implements WebGPUProgram {
     this.dispatchLayout = {x: [2], y: [1], z: [0]};
     this.dispatch = [
       Math.ceil(outputShape[2] / this.workGroupSize[0]),
-      Math.ceil(outputShape[1] * 2 / this.workGroupSize[1]), outputShape[0]
+      Math.ceil(outputShape[1] / this.workGroupSize[1]), outputShape[0]
     ];
```

Comment:
Nit: Please directly use
 `this.dispatch = computeDispatch( this.dispatchLayout, this.outputShape, this.workGroupSize);`